project/plugins.sbt

```diff
@@ -2,5 +2,6 @@ addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.2")
 addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.8.0")
 addSbtPlugin("com.github.cb372" % "sbt-explicit-dependencies" % "0.2.16")
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.1")
-addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.2.7")
+//addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.2.7")
```

Comment:
We still need it?

Reply:
I would like to leave it here for a while.
Most likely we will have to remove `coveralls` completely because it doesn't work well with `sbt`. But for now I want to explicitly show that we are aware about official plugin but it doesn't suite us.